In lieu of a hackathon (#155)
adobe/helix-home: showandtell/1-pumpkin-spice-show-and-tell.md

![](./1-pumpkin-spice-show-and-tell/1.jpg)

# Project Helix Show & Tell I

## The Pumpkin Spice Show & Tell

Oct 2020 9AM, Online

---

(See the [Archive](./README.md) for past Show & Tells – there aren't any, this will be the first one)

> Project Helix Show & Tells are on ongoing series of web meetings where Helix Pages users share the sites they have built with Helix, share feedback and inspiration with other users, and get tips & tricks from Project Helix developers.

Reviewer: ```suggestion
Project Helix Show & Tells are a quarterly series of web meetings where Helix users present the sites they have built with Helix, share feedback and inspiration with the group, and get tips & tricks from Project Helix developers.
```

### How it Works

> How will this work?

> We will three to four presenters for each show & tell. Presenters don't need slides, just a screen share of their site and a link to their GitHub repository.

Reviewer 2: ```suggestion
There will be three to four presenters for each show & tell. Presenters don't need slides, just a screen share of their site and a link to their GitHub repository.
```

## The Pumpkin Spice Show & Tell

> Oct 2020 9AM, Online

Author: ```suggestion
Oct 27th 2020, 9 AM PT, Online
```

#### Date and Time

> Oct 27 2020 9AM PT

Author: ```suggestion
Oct 27 2020
- San Jose: 9 am
- Basel: 5 pm
- Tokyo: 1 am (Oct 28th)
```
